source/fundamentals/atlas-vector-search.txt

```diff
@@ -0,0 +1,191 @@
+collection and perform vector search queries against vector data. You can also
+combine vector search with full-text search queries to return the most relevant
+results for your use case. You can integrate Atlas Vector Search with popular AI
+frameworks and services to easily implement vector search in your applications.
+
+To learn more about Atlas Vector Search, see the :atlas:`Atlas Vector Search
+</atlas-vector-search/vector-search-overview/>` guide in the MongoDB Atlas
+documentation.
+
+Supported Vector Embedding Types
+--------------------------------
+
+:atlas:`Vector Embeddings
+</atlas-vector-search/vector-search-overview/#std-term-vector-embeddings>`
+are vectors you use to represent your data. These embeddings
+capture meaningful relationships in your data and enable tasks like semantic
+search and retrieval.
+
+The {+driver-short+} supports vector embeddings of several types. The following
+sections describe the supported vector embedding types.
+
+
+.. _csharp-vector-array-representation:
+
+Array Representations
+~~~~~~~~~~~~~~~~~~~~~
+
+The {+driver-short+} supports the following representations of the array
+type in vector embeddings:
+
+- ``BsonArray``
+- ``Memory``
+- ``ReadOnlyMemory``
+- ``float[]`` and ``double[]``
+
+The following example shows a class with properties of the preceding types:
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-vector-search/VectorSearchExamples.cs
+   :language: csharp
+   :start-after: start-bson-arrays
+   :end-before: end-bson-arrays
+
+.. _csharp-binary-vector-representation:
+
+Binary Vector Representations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The {+driver-short+} supports the following binary vector representations in
+vector embeddings:
+
+- ``BinaryVectorFloat32`` (not supported on big-endian architectures)
+- ``BinaryVectorInt8``
+- ``BinaryVectorPackedBit``
+- ``Memory<float>``, ``Memory<byte>``, ``Memory<sbyte>``
+- ``ReadOnlyMemory<float>``, ``ReadOnlyMemory<byte>``, ``ReadOnlyMemory<sbyte>``
+- ``float[]``, ``byte[]``, ``sbyte[]``
+
+.. note::
+
+   You must use the ``BinaryVector`` attribute when specifying binary vector
+   representations of the ``Memory<T>``, ``ReadOnlyMemory<T>``, or array
+   types.
+
+The following example shows a class with properties of the preceding types:
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-vector-search/VectorSearchExamples.cs
+   :language: csharp
+   :start-after: start-binary-vectors
+   :end-before: end-binary-vectors
+
+Binary Vector Data Serialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can serialize ``Int8`` binary vector typed data as ``byte`` or ``sbyte``
+as shown in the following example:
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-vector-search/VectorSearchExamples.cs
+   :language: csharp
+   :start-after: start-binary-int-serialize
+   :end-before: end-binary-int-serialize
+
+You can deserialize ``PackedBit`` vector data to a :ref:`binary vector
```

Comment:
I think for completeness to clarity it's worth mentioning the supported .NET types for other vector types.
PackedBit -> byte
Float32 -> float

I think it's natural for users to expected built-in support for other types, which we might add in the future. So it's better to explicitly state what's supported now.

Reply:
Added float to the first example and updated the packedbit description to specify that it serializes to byte. Let me know if that's what you had in mind!